src/components/QuestionDisplay/AnswerButton.jsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from '@mui/material';
+
+const AnswerButton = ({ answer, isCorrect, showResult, onAnswer }) => {
+  const getColor = () => {
+    if (!showResult) return 'primary';
+    if (isCorrect) {
+      return 'success';
+    }
+    return 'error';
+  };
+
+  return (
+    <Button
+      variant="contained"
+      color={getColor()}
+      onClick={() => onAnswer(answer)}
```

Comment:
This only uses the `onAnswer` and `answer` from the props. If that's the case, I would pass the `() => onAnswer(answer)` as a prop instead, and don't pass the `onAnswer` prop.